src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/CourtAppearanceController.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
+import jakarta.persistence.EntityNotFoundException
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto.CreateCourtAppearance
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto.CreateCourtAppearanceResponse
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.service.CourtAppearanceService
+
+@RestController
+@RequestMapping("/courtAppearance", produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
Should these be kebab case? court-appearance

Reply:
sorry, yeah. they should!